ac.soton.eventb.emf.diagrams.refactor/META-INF/MANIFEST.MF

```diff
@@ -15,6 +15,7 @@ Require-Bundle: org.eclipse.emf.common.ui;bundle-version="[2.6.0,3.0.0)",
  ac.soton.eventb.emf.core.extension.navigator;bundle-version="[4.1.1,5.0.0)",
```

Comment:
```suggestion
 ac.soton.eventb.emf.core.extension.navigator;bundle-version="[5.0.0,6.0.0)",
```

Reply:
oops I forgot to commit that one. Have done it now.